docker-multinode/common.sh

```diff
@@ -199,7 +196,6 @@ kube::multinode::start_k8s_master() {
       --cluster-dns=10.0.0.10 \
       --cluster-domain=cluster.local \
       ${CNI_ARGS} \
-      --hostname-override=$(ip -o -4 addr list ${NET_INTERFACE} | awk '{print $4}' | cut -d/ -f1) \
```

Comment:
It doesn't work  without this parameter for multiple node cluster with the same hostname (environment with VMs). Only master is visible in cluster in that case.